source/client-side-encryption/tests/README.rst

```diff
@@ -371,6 +374,31 @@ In the prose tests LOCAL_MASTERKEY refers to the following base64:
 
 Perform all applicable operations on key vault collections (e.g. inserting an example data key, or running a find command) with readConcern/writeConcern "majority".
 
+Custom Key Material Test
```

Comment:
I didn't catch this earlier, but please make sure all prose tests in this file are numbered. By numbering them and not re-using numbers from retired tests, it's easier for drivers to keep prose tests in sync and track which have and have not been implemented.

See: https://github.com/mongodb/specifications#prose-test-numbering